### PR TITLE
Added Card: Natural Balance

### PR DESCRIPTION
**Pull request by bugfi5h**

### CODE REVIEW

**Comment by ingmargoudt** [comment]
Looks OK to me, can you fix the indentation? { on the same lines, and spacing between operators

**Comment by ingmargoudt** [comment]
if you are on intelliJ : press ctrl + l

**Comment by bugfi5h** [comment]
Like this?

File: Mage.Sets/src/mage/cards/n/NaturalBalance.java [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+        @Override
+        public NaturalBalanceEffect copy() {
+            return new NaturalBalanceEffect(this);
+        }
+
+        @Override
+        public boolean apply(Game game, Ability source) {
+            Player controller = game.getPlayer(source.getControllerId());
+            if (controller != null) {
+                PlayerList players = game.getState().getPlayerList(controller.getId());
```
Comment (JRHerlehy) [inline Mage.Sets/src/mage/cards/n/NaturalBalance.java]: Use getPlayersInRange to get only the affected players and not all players at the table.

**Comment by LevelX2** [comment]
Thanks for your contribution.
I updated some minor things 108f3faaa82a6e3e6597e68b9d0db6d79cd50ea5.

**Pull request merged**